_posts/2019-11-13-AWS-Dev-Day-2019.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+authors: [yolan_vloeberghs]
+title: 'AWS Dev Day 2019'
+image: /img/aws-dev-day-2019-banner.png
+tags: [AWS, Cloud, Conference]
+category: Conference
+comments: true
+---
+# AWSome Dev Day 2019!
+
+>The AWS Dev Day is a free conference day hosted by Amazon where architects and developers from Amazon and other companies talk about the most popular AWS services, the newest features and everything related to cloud development. These sessions explain a lot about what AWS has to offer and you will see live demos on how you can implement them in your own use cases. 
+
+This year, AWS decided to host their annual Dev Day conference in Belgium, more specifically Mechelen. This was very convenient us, as our headquarters are located in Mechelen aswell. This means that there was a very good reason for Ordina consultants to attend the event. We went with a small group of 8 people to make sure we didn't miss the latest AWS news. We were treated with delicious food & drinks throughout the day to make sure their guests were happy.
+
+# Table of contents
+
+
+# CI/CD for Modern Applications by Sebastian Stormacq
+
+### IAC
+In an application development environment, it is important to maintain your Infrastructure as Code (IaC) to allow for repeatibility, and doing so in a declarative way. Doing it in an imperative way does not allow for deltas and requires your system to be rebuilt from scratch every time your pipeline gets triggered, which results in worse impact for your end users.
+
+There are a number of ways to provide your infrastructure as a code declaratively. We have Terraform, Cloudformation, etc. But Amazon has a new kid on the block since this year: the [Cloud Development Kit (CDK)](https://aws.amazon.com/cdk/){:target="_blank" rel="noopener noreferrer"}. This allows developers to provide your infrastructure in your favorite programming language. You can communicate with AWS in a different way other than the AWS CLI or the Web Console, as you can now create services as you need them and when you need them.
```

Comment:
```suggestion
There are a number of ways to provide your infrastructure as a code declaratively. We have Terraform, Cloudformation, etc. But Amazon has a new kid on the block since this year: the [Cloud Development Kit (CDK)](https://aws.amazon.com/cdk/){:target="_blank" rel="noopener noreferrer"}. This allows developers to provide your infrastructure in their favorite programming language. You can communicate with AWS in a way other than the AWS CLI or the Web Console, as you can now create services as you need them and when you need them.
```